decavision/model_training/tfrecords_image_classifier.py

```diff
@@ -298,21 +298,32 @@ def fit(self, save_model=None, export_model=None, min_accuracy=None,
             verbose (bool): show details of training or not
             fine_tuning (bool): fine tune pretrained model or not
             l2_lambda (float): amount of L2 regularization to include in extra layer
-            min_accuracy (float): if specified, stop training when improvement in val accuracy is smaller than min_accuracy
+            patience (int): if specified, stops training when improvement in val accuracy is not observed using early stopping
             logs (str): if specified, tensorboard is used and logs are saved at this location
         """
         callbacks = None
```

Comment:
I don't think this line is useful anymore since callbacks is always defined by the lines below.